sdk/monitor/azure-monitor-query/src/main/java/com/azure/monitor/query/LogsQueryClientBuilder.java

```diff
@@ -33,7 +47,7 @@
  * </pre>
  * <!-- end com.azure.monitor.query.LogsQueryAsyncClient.instantiation -->
  *
- * <p><strong>Instantiating a synchronous Logs query Client</strong></p>
+ * <p>The following sample shows instantiating a synchronous Logs query Client using Token Credential</p>
```

Comment:
`Client` -> `client` and `using Token Credential` should be updated to `using {@link TokenCredential}`